pkg/client/observatorium/observability_config.go

```diff
@@ -118,11 +120,29 @@ func (c *ObservabilityCloudWatchLoggingConfig) setDefaults() {
 	}
 }
 
+func (c *ObservabilityCloudWatchLoggingConfig) GetEnterpriseCredentials(orgID string) *ObservabilityCloudwatchLoggingConfigCredentials {
+	credentials := ObservabilityCloudwatchLoggingConfigCredentials{}
+	idx, enterpriseCredential := arrays.FindFirst(c.EnterpriseCredentials, func(enterpriseCredential ObservabilityEnterpriseCloudwatchLoggingConfigCredentials) bool {
+		return orgID == enterpriseCredential.OrgID
+	})
+	if idx != arrays.ElementNotFound {
+		credentials.AccessKey = enterpriseCredential.Credentials.AccessKey
+		credentials.SecretAccessKey = enterpriseCredential.Credentials.SecretAccessKey
+	}
+
+	return nil
+}
```

Comment:
This function always returns `nil`

Reply:
> This function always returns nil.

I think this is the reason the test are failing as well

Reply:
Yeah I just caught this, fixing now